app/assets/javascripts/utilities/buildCommentFormHTML.js.erb

```diff
@@ -42,7 +42,7 @@ function buildCommentFormHTML(commentableId, commentableType, parentId) {
         <div class="response-templates-container crayons-card crayons-card--secondary p-4 mb-4 fs-base comment-form__templates hidden">
           <header>
             <button type="button" class="personal-template-button active" data-target-type="personal" data-form-id="new_comment">Personal</button>
-            <button type="button" class="moderator-template-button hidden" data-target-type="moderator" data-form-id="new_comment">Moderator</button>
+            <button type="button" class="moderator-template-button hidden" data-target-type="moderator" data-form-id="new_comment">Trusted User</button>
```

Comment:
@msarit I want to make sure we're getting our nomenclature aligned.  We're shifting "Moderator" label to "Trusted User" but the underlying role name is the same.

This is setting my Spidey senses tingling.

Reply:
From what I recall, that's correct. The [role that I added was named Moderator](https://github.com/forem/forem/pull/17737/files#diff-8302388bad5bdfb6e929ae303c2ec4daed3f73e18cee56d48dc7436867c942aeR8)

Reply:
@jaw6 @dscottS3 So let me get this straight: we are giving this ability to the Moderator Role, but we are calling them "Trusted Users"? 🤔

Reply:
Slow down.

We have work to add a new role called `moderator` which is what @dscottS3 and @msarit are working on.  That is a brand new role that sits between `tag moderator` and `admin` and allows a `moderator` to suspend/unpublish and manage all tags.

@jaw6 is working on taking the existing `mod_comment` template and moving it from only working with the `tag moderator` role to the `trusted user` role.  Given we are moving who can access it, we should also change the name of the tab in the UI too.  We aren't giving this ability to the `moderator` role.  We are giving it to the `trusted user` role.

I think the rest of @jaw6 's comment about naming is a technical concern for those in the code.  I agree it is confusing to use the term `mod_comment` and the form of `moderator` given that the actual role that will have access is `trusted user`.

Did that help?

Reply:
@jaw6 @JennieOcken Thank you for these explanations. 🙏🏾  I'm clear now.